content/articles/url-record.markdown

```diff
@@ -25,6 +25,15 @@ Enter the hostname you would like to redirect from, and the URL to redirect to.
 
 Right now it is not possible for our redirection service to handle SSL requests for your domain names. Take a look at the article [Can a URL record redirect requests over SSL?](/articles/url-redirect-ssl) for additional information and an explanation of alternatives.
 
-## URL Rewrites
+## URL to URL redirects
+
+URL to URL redirects can't be done with our URL record. Only domains or subdomains can be redirected to complete URLs. 
+
+Example: foo.com/blog to bar.com/blog
```

Comment:
Maybe add a note that the desired behavior (that's not possible with DNS) is to only redirect foo.com/blog, but not foo.com or foo.com/something-else